common.gypi

```diff
@@ -79,8 +79,8 @@
 
     'conditions': [
       ['target_arch=="arm64"', {
-        # Disabled pending https://github.com/nodejs/node/issues/23913.
-        'openssl_no_asm%': 1,
+        # Enable openssl asm for arm64, on a test basis only.
+        'openssl_no_asm%': 0,
```

Comment:
The two branches of the conditional are identical, they set `no_asm` to `0`, so the conditional can be removed, as well as the comment. Anyone wanting to know the history of when/why this was changed can use `git blame`, which will lead to the commit, which will lead to this PR and its discussion.

Reply:
Conditional was removed, but that value still needs to be set somewhere.